Source/ui_libretro/main_libretro.cpp

```diff
@@ -493,6 +493,19 @@ bool retro_load_game(const retro_game_info* info)
 {
 	CLog::GetInstance().Print(LOG_NAME, "%s\n", __FUNCTION__);
 
+#if defined(IOS)
+	bool can_jit;
```

Comment:
Could we default init this to `false` just to be sure?

Reply:
Yes of course, I've updated the PR with that change.